sceptre/cli/__init__.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+# flake8: noqa
```

Comment:
Why's this required?

Reply:
Actually I could avoid falling foul of the complexity check by defining my function `deep_merge` outside of the `cli` function. I don't think that leads to improvement in the code however but it is an option.

Reply:
It would be good to move it out into some sort of utilities file. Makes it easier to test too